src/components/SearchView.tsx

```diff
@@ -86,25 +87,40 @@ export class SearchView extends React.Component<SearchViewProps, SearchViewState
     }
 
     onTextChange(event: any) {
+        clearTimeout(this.timeout);
```

Comment:
Would you think we need to set this value to an invalid number (e..g `-1`) after clearing the timer? The same question can be asked in constructor too (e.g. `timeout: number = -1`)?